### DISCUSSION TITLE
ChatGPT Codeinterrupter生成的文件无法识别下载，会识别成链接打开，导致乱码。

### DISCUSSION THREAD

**Discussion by indincys** [tw93/Pake#526] [Q&A]
![-ChatGPT-07-22-08 12 19](https://github.com/tw93/Pake/assets/14246072/c378377d-4261-4187-af6a-2280ffde9b37)

**Comment by tw93** [comment]
谢谢反馈，我复现了，我修复一下

**Comment by tw93** [comment]
已经修复，可以下载最新的版本
